static/skin/iso6391To3.js

```diff
@@ -1,85 +1,134 @@
 // eslint-disable-next-line no-unused-vars
 const iso6391To3 = {
```

Comment:
Please put the source of this list, if you have choice, please try to find a source of reference (unicode consortium or reference implementation).

Reply:
Used this https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes